.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
```

Comment:
will this file be picked up automatically by pre-commit?

Reply:
Yes, it is a configuration file which is automatically detected at root of the repo.